src/app/components/Rating/rating.css

```diff
@@ -0,0 +1,11 @@
+.star {
```

Comment:
You don't need this native css file, you can use module css for everything what you have